<launch>
    <!-- Start piCam -->
    <!--include file="$(find usb_camera)/launch/logi_cam_640x480.launch"/-->

    <node pkg="aruco_ros" type="marker_publisher" name="aruco_marker_publisher" required="false" output="screen">
        <remap from="/camera_info" to="/usb_cam/camera_info" />
        <remap from="/image" to="/usb_cam/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value=".05"/>
        <param name="reference_frame"    value="cam_frame"/>
        <param name="camera_frame"       value="cam_frame"/>
        <param name="tag_config_json"    value="$(find magdrone)/cfg/aruco_bundlev3.json"/>
    </node>

</launch>
